samples/system-test/detect.test.js

```diff
@@ -297,3 +298,27 @@ test(`should extract text from pdf file`, async t => {
   );
   t.true(output.includes('pdf-ocr.pdf.json'));
 });
+
+test(`should detect objects in a local file`, async t => {
+  const output = await tools.runAsync(
+    `${cmd} localize-objects ${files[8]}`,
```

Comment:
Test fails – the command run is`$ node detect.js localize-objects [object Object]`

Change `files[8]` to `files[8].localPath`